fs/fs.go

```diff
@@ -282,7 +259,7 @@ func (self *RealFsInfo) GetDirFsDevice(dir string) (*DeviceInfo, error) {
 }
 
 func (self *RealFsInfo) GetDirUsage(dir string) (uint64, error) {
-	out, err := exec.Command("du", "-s", dir).CombinedOutput()
+	out, err := exec.Command("/bin/sh", "-c", "nice -n 19 du -s", dir).CombinedOutput()
```

Comment:
Probably don't need to wrap it in `/bin/sh` now as it's not a binary, not a shell builtin. This will also help with the args - `dir` is outside of the `-c` param to `sh` right now & probably won't be used at all in this call.

I'd go with `out, err := exec.Command("nice", "-n", "19", "du", "-s", dir).CombinedOutput()`

Unless I'm missing it, could you add a test for this too? Should be pretty easy to set up a file with known file in an otherwise empty dir & run `GetDirUsage` on it to check.